test/helpers/reactjs_helper_test.rb

```diff
@@ -39,4 +39,13 @@ def webpack_asset_paths(bundle_name, opts)
     assert res.include?('webpack/foreman_react.js')
     assert res.include?('webpack/foreman_angular.js')
   end
+
+  test "should be able to load global js in foreman core" do
+    Foreman::Plugin.register :plugin_with_global_js do
+      register_global_file 'some_global_file'
```

Comment:
Can we name this `register_global_js_file` so it's more clear what this is used for?